install.sh

```diff
@@ -83,6 +83,16 @@ configure() {
     args=""
     add_arg() {
         log_debug "Add arg -$1=$2"
+        multiline=$(is_multiline "$2")
+        if [ "$multiline" -eq "1" ]; then
```

Comment:
Would be easier to follow if `is_multiline` was returning a success status if multiline instead of printing it. This condition could then be: `if is_multiline "$2"; then`.